kite-api/src/express.api.js

```diff
@@ -2,9 +2,10 @@ require('dotenv').config();
 const express = require('express');
 const session = require('express-session');
 
+const { ApolloServer, gql } = require('apollo-server-express');
+
 const redis = require('redis').createClient(6379, 'kiteredis');
 const RedisStore = require('connect-redis')(session);
-const cors = require('cors');
```

Comment:
Why was CORS removed? Does it work without it now?

Reply:
Apollo includes its own cors library

Reply:
😎😎😎